Workflow/Modules/ModuleBase.py

```diff
@@ -357,7 +357,7 @@ def setApplicationStatus( self, status, sendFlag = True ):
       # The application status won't be updated in case the workflow or the step is failed already
       if not isinstance( status, basestring ):
         status = str( status )
-      self.log.verbose( 'setJobApplicationStatus(%d, %s)' % ( self.jobID, status ) )
+      self.log.verbose( 'setJobApplicationStatus(%s, %s)' % ( self.jobID, status ) )
```

Comment:
This is probably the DIRAC version in the pilot was v6r17-pre3 and not the latest one. In any case this change will work both for str and int. So, this will not harm.